sql/sqlite/migrate_test.go

```diff
@@ -411,6 +411,58 @@ func TestPlanChanges(t *testing.T) {
 				},
 			},
 		},
+		// Table options.
+		{
+			changes: []schema.Change{
+				// Create a new table with options.
+				&schema.AddTable{
+					T: schema.NewTable("t1").
+						SetSchema(schema.New("s1")).
+						AddColumns(schema.NewIntColumn("id", "int")).
+						AddAttrs(&WithoutRowID{}, &Strict{}),
+				},
+				// Modify options for an existing table.
+				func() schema.Change {
+					t2 := schema.NewTable("t2").
+						SetSchema(schema.New("s2")).
+						SetPrimaryKey(schema.NewPrimaryKey(schema.NewIntColumn("id", "int"))).
+						AddAttrs(&WithoutRowID{})
+					return &schema.ModifyTable{
+						T: t2,
+						Changes: []schema.Change{
+							&schema.AddAttr{
+								A: &WithoutRowID{},
+							},
+							&schema.DropAttr{
+								A: &Strict{},
+							},
```

Comment:
Great, but since there's no special code for table options modification, this case be deleted IMO.